### PR TITLE
Jmax/lg 8870 update vendor outage runbook

### PR DESCRIPTION
**Pull request by jmax-gsa**
Updated section on Identity Verification outage flags.

### CODE REVIEW

**Review by zachmargolis — APPROVED** [review]
LGTM, I would also add @18F/identity-ada as reviewers too? Since that's the team with more context on these changes?

File: _articles/vendor-outage-response-process.md [hunk 1]
```diff
@@ -60,26 +60,88 @@ The possible values for each flag:
...
+
+ - The `SMS` checkbox on the IdV dialogs will be disabled.
+ 
+ - New phone numbers for SMS 2FA will not be verified.
+
+  - Any user entering the verification flow will be presented with an
+    outage screen telling them that their options are to verify by
+    mail or wait until our vendor outage is resolved.
+
+  - Users will only be able to upload their IDs from their computer;
```
Comment (benjaminchait) [inline _articles/vendor-outage-response-process.md:134]: Minor nit: Users who _start_ on a mobile phone will be okay, but the _hybrid handoff_ will not be available.

What do you think of: "Users will need to add photos of their ID from the device on which they start; desktop users will not be presented the opportunity to switch to a mobile phone for adding their ID."

File: _articles/vendor-outage-response-process.md [hunk 2]
```diff
@@ -60,26 +60,88 @@ The possible values for each flag:
...
+#### Voice
+  - Users will only be able to verify their identity by mail.
+  
+  - New phone numbers for voice 2FA will not be verified
+ 
+  - Any user entering the verification flow will be presented with an
+    outage screen telling them that their options are to verify by
+    mail or wait until our vendor outage is resolved.
+
+  - Users will only be able to upload their IDs from their computer;
```
Comment (benjaminchait) [inline _articles/vendor-outage-response-process.md:146]: I don't think we use the Pinpoint "voice" feature for hybrid handoff, so I think this line could be removed? 

But all other content in this section makes sense!

**Review by benjaminchait — CHANGES_REQUESTED** [review]

File: _articles/vendor-outage-response-process.md (hunk 1 above)
Comment (jmax-gsa) [reply _articles/vendor-outage-response-process.md:134]: Re-worded your suggestion a bit; see if you like it.

File: _articles/vendor-outage-response-process.md (hunk 2 above)
Comment (jmax-gsa) [reply _articles/vendor-outage-response-process.md:146]: Nope. That's how the system behaves right now. I'm on board with changing it; I'll even write the ticket if you like. But right now, we don't offer hybrid handoff if voice is disabled.

The spec at `features/idv/outage_spec:185` check for exactly this behavior, and I just manually verified it to be extra doubleplus sure.

Comment (benjaminchait) [reply _articles/vendor-outage-response-process.md:146]: Ah, that is...surprising!

Yes, can you drop a ticket on Team Ada's board, and we can refine/update things accordingly?

**Pull request closed without merging**